src/pixelHuntStudy.ts

```diff
@@ -99,8 +99,8 @@ export async function pageDataListener(pageData) {
     }
     const pageVisitStart = new Date(pageData.pageVisitStartTime);
```

Comment:
I had the patch a little wrong here - `pageData` props are not changing, just the glean-generated `userJourney` functions below.